src/main/java/io/codelex/classesandobjects/practice/CarChanged.java

```diff
@@ -2,28 +2,34 @@
 
 public class CarChanged {
 
-    double startKilometers;
-    double liters;
-    double currentKilometers;
+    private static final double CONS_5 = 5.0;
```

Comment:
šo domāju vairāk kā ar vārdiem, piemēram, `MAX_ECONOMY_CONSUMTION` something like that

Reply:
Izlaboju. Paldies, nezināju kā labāk nosaukt..